eye2bids/config/eyelink_metadata.yml

```diff
@@ -10,15 +10,15 @@
 SampleCoordinateUnits: str
 SampleCoordinateSystem: str
 EnvironmentCoordinates: str
-ScreenDistance: int
+ScreenDistance: float # or express as X,Y,Z coordinates, e.g. [0.05, 0.30, 0.40]
```

Comment:
I am trying to use this tool to generate a BIDS compliant dataset for one subject.

The thing is that Eyetracking was collected for this subject using EyeLink's _remote mode_, so I actually don't actually have an accurate or stable estimation of distance.

Also - According to the BIDS spec - this field isn't actually required?

Should this field be removed from the required section?

Reply:
Ah yes, sorry, I thought it was required at some point...I'll change this!